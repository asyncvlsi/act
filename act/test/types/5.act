defchan dualrail <: chan(bool) (bool d0,d1,de; bool d[2])
{
  d0 = d[0];
  d1 = d[1];
  spec {
  
  }
}

defproc foo (dualrail l, r)
{

}

defproc bar (foo p, q)
{

}
