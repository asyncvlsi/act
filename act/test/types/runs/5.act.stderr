ERROR: File `5.act', line 15, col 19
	Parameter `p': port parameter for a process cannot be a process
